### DISCUSSION TITLE
Stories not working - 404

### DISCUSSION THREAD

**Discussion by gapetto** [reactivemarkets/react-financial-charts#592] [General]
Stories not working - 404 on https://reactivemarkets.github.io/react-financial-charts/

**Comment by markmcdowell** [comment]
You’re using the wrong url:

https://react-financial.github.io/react-financial-charts/

**Reply by gapetto** [reply]
Thx!

**Reply by gapetto** [reply]
It's on your repo though: https://github.com/react-financial/react-financial-charts/tree/64e893fb3a2d1c2b3f430b90572784f5c8fe02fa

**Reply by gapetto** [reply]
It's not, sorry wrong branch

**Reply by markmcdowell** [reply]
Yeah old code on that commit
